packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/FormStep/FormStepWithFields/FormStepRow.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback, useMemo } from "react";
+import { FbFormModelField, FbFormStep } from "~/types";
+import Draggable, { BeginDragProps } from "~/admin/components/FormEditor/Draggable";
+import { RowHandle, Row, RowContainer } from "../../Styled";
+
+import { Icon } from "@webiny/ui/Icon";
+import { ReactComponent as HandleIcon } from "~/admin/components/FormEditor/icons/round-drag_indicator-24px.svg";
+import { Horizontal } from "~/admin/components/FormEditor/DropZone";
+import { useFormStep } from "~/admin/components/FormEditor/Tabs/EditTab/FormStep/useFormStep";
+import { DragObjectWithFieldInfo } from "~/admin/components/FormEditor/Droppable";
+import { FormStepRowField } from "./FormStepRowField";
+
+export interface FormStepRowProps {
+    formStep: FbFormStep;
+    rowIndex: number;
+    row: FbFormModelField[];
+    isLastRow: boolean;
+}
+
+export const FormStepRow = (props: FormStepRowProps) => {
+    const { formStep, rowIndex, row, isLastRow } = props;
+
+    const { onFormStepDrop, handleDrop } = useFormStep();
```

Comment:
But we are not using it directly. It is being passed as a dependency to the `useCallback` so it would work properly

Reply:
The suggestion was to merge `handleDrop` and ` onFormStepDrop` since `handleDrop` is not being used anywhere. By doing that, we'll be able to have `onFormStepDrop` as a dep.

Does that make sense?

Reply:
Yeah, make sence. In the latest commit I have merged those two methods.